src/statbank/uttrekk_validations.py

```diff
@@ -642,4 +741,6 @@ def _check_rounding_per_variable(
                 ValueError(error_text)
             )
             logger.warning(error_text)
+        del data_to_validate
+        gc.collect()
```

Comment:
Skjønner du er redd for at brukeren går tom for minne når du lager en kopi av alle tabellene, men skeptisk til at et kall til gc.collect() etterpå har noe for seg. Python er ikke et språk hvor manuell minnehåndtering er nødvendig, objekter blir fjernet automatisk når de går ut av «scope», og kall til gc.collect() kan føre til at programmet kjører saktere.

Reply:
Good feedback, removing gc.collect() from the validation code. Need a new review though.